server/src/optima/utils.py

```diff
@@ -16,17 +16,17 @@ def check_project_name(api_call):
     @wraps(api_call)
     def _check_project_name(*args, **kwargs):
```

Comment:
would be nice to rename this decorator, as it is not only checking for the name anymore